package.json

```diff
@@ -26,7 +26,8 @@
     "build": "gulp build",
     "test": "gulp test",
     "cf-link": "node scripts/npm/link",
-    "postinstall": "npm run cf-link"
+    "postinstall": "npm run cf-link",
+    "process-icon-svgs": "svgo -o src/cf-icons/src/icons --enable=addClassesToSVGElement --enable=removeStyleElement --enable=removeAttrs --config='{ \"plugins\": [ { \"addClassesToSVGElement\": { \"className\": \"cf-icon-svg\" } }, { \"removeAttrs\": { \"attrs\": [ \"fill\", \"path:class\", \"circle:class\" ] } }, { \"cleanupIDs\": { \"force\": \"true\" } } ] }'"
```

Comment:
What calls this? I don't see any other reference to this script?

Reply:
It's called manually when someone adds/updates icons. We need to add documentation about that somewhere.